src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Translation;
 
+@trigger_error('The '.__NAMESPACE__.'\Translator class is deprecated since version 2.8 and will be removed in 3.0. Use directly the Symfony\Component\Translation\Translator class instead.', E_USER_DEPRECATED);
```

Comment:
Removing a class from `FrameworkBundle` for one from the component is always great news \o/